pykob/morse.py

```diff
@@ -274,3 +327,9 @@ def lookupChar(self, code):
             return(decodeTable[codeTableIndex][code])
         else:
             return('')
+
+    def displayBuffers(self, text):
+        """Display the code buffer and other information for troubleshooting"""
+        print(text, "nChars", self.nChars)
```

Comment:
I would like to use the `pykob.log.debug` here. For now it simply prints with a 'debug' label and a timestamp. But, the goal for the future is that these can be externally enabled and disabled.

Reply:
Great suggestion. I've made the change locally, and will push the updated branch and create a new pull request once I've received feedback from @jchausler.